### PR TITLE
Add vila mariana

### PR DESCRIPTION
**Pull request by rhnonose**
- Add a task to insert new district
- Add some normalization for district aliases
- Add some tests for the task, mainly to use as a future reference for tasks since we might be using more frequently in favor of migrations

### CODE REVIEW

File: apps/re/lib/addresses/neighborhoods.ex [hunk 1]
```diff
@@ -89,9 +89,43 @@ defmodule Re.Addresses.Neighborhoods do
 
   defp normalize(
          %{city_slug: "sao-paulo", state_slug: "sp", neighborhood_slug: "pompeia"} = neighborhood
-       ) do
-    %{neighborhood | neighborhood_slug: "vila-pompeia", neighborhood: "Vila Pompeia"}
-  end
+       ),
+       do: %{neighborhood | neighborhood_slug: "vila-pompeia", neighborhood: "Vila Pompeia"}
+
+  defp normalize(
```
Comment (pmargreff) [inline apps/re/lib/addresses/neighborhoods.ex]: A suggestion here, in these cases`remap_neighborhood`, `reroute_neighborhood`, `rematch_neighborhood` or something on this direction sounds meaningful than `normalize`

File: apps/re/lib/mix/tasks/addresses/add_district.ex [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        state: "SP",
+        city: "São Paulo",
+        status: "active"
+      })
+      |> Re.Repo.insert()
+
+    Logger.info("Inserted district named #{district.name}")
+  end
+
+  def insert(params) do
```
Comment (pmargreff) [inline apps/re/lib/mix/tasks/addresses/add_district.ex]: Is this fn right?

Comment (rhnonose) [reply apps/re/lib/mix/tasks/addresses/add_district.ex]: No, copy-paste problems 😅

**Comment by rhnonose** [comment]
The districts are inserted as is, this is just to check if it's covered or not.
We definitely should check the "browseability" of those aliases for ssure.

File: apps/re/lib/addresses/district.ex [hunk 3]
```diff
@@ -21,8 +21,8 @@ defmodule Re.Addresses.District do
     timestamps()
   end
 
-  @required ~w(state city name description)a
-  @optional ~w(status)a
+  @required ~w(state city name)a
+  @optional ~w(status description)a
```
Comment (pmargreff) [inline apps/re/lib/addresses/district.ex]: Couldn't this cause a regression on the client?

Comment (rhnonose) [reply apps/re/lib/addresses/district.ex]: What kind of regression?

Comment (pmargreff) [reply apps/re/lib/addresses/district.ex]: An `undefined` on neighborhood description.

Comment (rhnonose) [reply apps/re/lib/addresses/district.ex]: Our frontend removed recently, but we might send a default empty string just in case, I'll check with them.

**Pull request merged**